.github/scripts/trigger-and-wait.sh

```diff
@@ -87,11 +87,14 @@ trigger_workflow() {
     -H "Content-Type: application/json" \
     -H "Authorization: Bearer ${INPUT_GITHUB_TOKEN}" \
     --data "{\"ref\":\"${ref}\",\"inputs\":${inputs}}"
+  
+  # Sleep after triggering workflow so it can be polled for status
+  echo "Sleeping for $wait_interval seconds"
+  sleep $wait_interval
```

Comment:
Issue was at parent workflow where a commit seems to have removed this wait time ( [Line #88-89](https://github.com/convictional/trigger-workflow-and-wait/pull/7/files#) in the original repo PR)

Adding back the sleep time post triggering for polling workflow status.